libraries/AP_TECS/AP_TECS.h

```diff
@@ -77,6 +77,13 @@ class AP_TECS {
         return _TAS_dem_adj / _ahrs.get_EAS2TAS();
     }
 
+    // Return stall airspeed on level flight. This is based on AIRSPEED_MIN as a
+    // very conservative estimate if AIRSPEED_STALL is not set.
+    float get_stall_airspeed_1g(void) const {
```

Comment:
Is there any reason this needs to be in TECS? Plane can see these parameters directly.

Reply:
I can't find it anywhere else?

Reply:
Remove for now, simple PRs get in easier :)